diff --git a/regulations/urls.py b/regulations/urls.py
index c66d546b..24521215 100644
--- a/regulations/urls.py
+++ b/regulations/urls.py
@@ -1,4 +1,6 @@
+from django.conf import settings
 from django.conf.urls import url
+from django.conf.urls.static import static
 
 from regulations.url_caches import daily_cache, lt_cache
 from regulations.views.about import about
@@ -179,4 +181,4 @@ urlpatterns = [
     url(r'^partial/%s/%s$' % (paragraph_pattern, version_pattern),
         lt_cache(PartialParagraphView.as_view()),
         name='partial_paragraph_view'),
-]
+] + static("/static/", document_root=settings.STATIC_ROOT)
